sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveCompatibilitySuite.scala

```diff
@@ -221,9 +221,6 @@ class HiveCompatibilitySuite extends HiveQueryFileTest with BeforeAndAfter {
     "udf_when",
     "udf_case",
 
-    // Needs constant object inspectors
-    "udf_round",
```

Comment:
Yes, I mean we can change that after #6835 merged, not now. Currently, seems only enable `udf_round_3` is enough, right? I'll agree if you update the comment to this `udf_round` in the black list, as it's not correct.